### PR TITLE
Don't re-export imports outside of `__init__.py`

### PR DESCRIPTION
**Pull request by Avasam**
And check for conventions regressions with Ruff

### CODE REVIEW

File: adodbapi/process_connect_string.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """ a clumsy attempt at a macro language to let the programmer execute code on the server (ex: determine 64bit)"""
 
-from . import is64bit as is64bit
+from . import is64bit
```
Comment (Avasam) [inline adodbapi/process_connect_string.py:3]: @vernondcole I try to keep adodbapi changes in separate PRs, but this one had only a single line that has no effect on runtime. This is purely semantics for static tools and checkers (so a user will be told to prefer `from adodbapi import is64bit` instead of `from adodbapi.process_connect_string import is64bit`)

File: ruff.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+target-version = "py37"
+
+[lint]
+select = [
+    "PLC", # Pylint Convention
+]
+
+[lint.per-file-ignores]
+# Explicit re-exports is fine in __init__.py, still a code smell elsewhere.
+"__init__.py" = ["PLC0414"]
```
Comment (Avasam) [inline ruff.toml:11]: https://github.com/astral-sh/ruff/issues/6294

**Review by mhammond — APPROVED** [review]

**Comment by Avasam** [comment]
@vernondcole You resolved the comment, I assume that means you're ok with this change, but would like to confirm this has your approval.
